docs/source/notebooks/climex.ipynb

```diff
@@ -101,12 +101,14 @@
     }
    ],
    "source": [
-    "import xarray as xr\n",
     "import intake\n",
+    "import xarray as xr\n",
     "import xclim\n",
     "from dask.diagnostics import ProgressBar\n",
     "\n",
-    "cat = intake.open_esm_datastore(\"https://pavics.ouranos.ca/catalog/climex.json\")  # TEST_USE_PROD_DATA\n",
+    "cat = intake.open_esm_datastore(\n",
+    "    \"https://pavics.ouranos.ca/catalog/climex.json\"\n",
+    ")  # TEST_USE_PROD_DATA\n",
```

Comment:
Will break test as well, `# TEST_USE_PROD_DATA` not on same line.

Reply:
Gotcha